app/src/main/java/Bamboo/view/MenuBar.java

```diff
@@ -0,0 +1,36 @@
+package Bamboo.view;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class MenuBar extends JMenuBar{
+    private JMenu fileMenu;
```

Comment:
No space between class and field variables -> harder to see difference that it should be